python/HeaderService/hslib_salobj.py

```diff
@@ -0,0 +1,495 @@
+# This file is part of HeaderService
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import os
+import sys
+import socket
+import asyncio
+import time
+import types
+from . import hutils
+from . import hscalc
+from lsst.ts import salobj
+
+
+class HSWorker(salobj.BaseCsc):
+
+    """ A Class to run and manage the Header Service"""
+
+    def __init__(self, **keys):
+
+        # Load the configurarion
+        self.conf = types.SimpleNamespace(**keys)
```

Comment:
Thank you for making this change. Please consider naming this `self.config` as when you make the header service a ConfigurableCsc that is what the argument to the configure method will be called. (And it matches are usage in other CSCs).

Reply:
Sure, will change to self.config